docs/references/backend/user/get-user-list.mdx

```diff
@@ -15,16 +17,19 @@ function getUserList(): (params: UserListParams) => Promise<PaginatedResourceRes
-| `externalId?` | `string[]` | The external ID's to filter by. |
+| `limit?` | `number` | The number of results to return. Must be an integer greater than zero and less than 501. Can be used for paginating the results together with `offset`. Defaults to `10`. |
+| `offset?` | `number` | Skip the first `offset` results when paginating. Needs to be an integer greater or equal to zero. To be used in conjunction with `limit`. Defaults to `0`. |
+| `orderBy?` | `'created_at' | 'updated_at' | 'email_address' | 'web3wallet' | 'first_name' | 'last_name' | 'phone_number' | 'username' | 'last_active_at' | 'last_sign_in_at'` | Return users in a particular order. Prefix with a `-` to reverse the order. Prefix with a `+` to list in ascending order. Defaults to `'-created_at'`. |
+| `emailAddress?` | `string[]` | Filters users with the specified email addresses. Accepts up to 100 email addresses. Any email addresses not found are ignored. |
+| `phoneNumber?` | `string[]` | Filters users with the specified phone numbers. Accepts up to 100 phone numbers. Any phone numbers not found are ignored. |
+| `externalId?` | `string[]` | Filters users with the specified external IDs. For each external ID, the `+` and `-` can be prepended to the ID, which denote whether the respective external ID should be included or excluded from the result set. Accepts up to 100 external IDs. Any external IDs not found are ignored. |
+| `username?` | `string[]` | Filters users with the specified usernames. Accepts up to 100 usernames. Any usernames not found are ignored. |
+| `web3Wallet?` | `string[]` | Filters users with the specified web3 wallet addresses. Accepts up to 100 web3 wallet addresses. Any web3 wallet addressed not found are ignored. |
+| `userId?` | `string[]` | Filters users with the user IDs specified. For each user ID, the `+` and `-` can be prepended to the ID, which denote whether the respective user ID should be included or excluded from the result set. Accepts up to 100 user IDs. Any user IDs not found are ignored. |
+| `organizationId?` | `string[]` | Filters users that have memberships to the given organizations. For each organization ID, the `+` and `-` can be prepended to the ID, which denote whether the respective organization should be included or excluded from the result set. Accepts up to 100 organization IDs. |
+| `query?` | `string` | Filters users that match the given query. For possible matches, we check the email addresses, phone numbers, usernames, web3 wallets, user IDs, first and last names. The query value doesn't need to match the exact value you are looking for, it is capable of partial matches as well. |
+| `last_active_at_since` | `number` | Filters users that had session activity since the given date, with day precision. Providing a value with higher precision than day will result in an error. Example: use `1700690400000` to retrieve users that had session activity from 2023-11-23 until the current day. For example: `1700690400000` |
```

Comment:
```suggestion
| `last_active_at_since` | `number` | Filters users that had session activity since the given date, with day precision. Providing a value with higher precision than day will result in an error. Example: use `1700690400000` to retrieve users that had session activity from 2023-11-23 until the current day. For example: `1700690400000`. |
```